src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.T.cs

```diff
@@ -452,373 +379,1635 @@ public static T Max<T>(ReadOnlySpan<T> x)
+        /// </para>
+        /// </remarks>
+        public static void Hypot<T>(ReadOnlySpan<T> x, ReadOnlySpan<T> y, Span<T> destination)
+            where T : IRootFunctions<T> =>
+            InvokeSpanSpanIntoSpan<T, HypotOperator<T>>(x, y, destination);
+
+        /// <summary>Computes the element-wise remainder of the numbers in the specified tensors.</summary>
+        /// <param name="x">The first tensor, represented as a span.</param>
+        /// <param name="y">The second tensor, represented as a span.</param>
+        /// <param name="destination">The destination tensor, represented as a span.</param>
+        /// <exception cref="ArgumentException">Length of <paramref name="x" /> must be same as length of <paramref name="y" />.</exception>
+        /// <exception cref="ArgumentException">Destination is too short.</exception>
+        /// <exception cref="ArgumentException"><paramref name="x"/> and <paramref name="destination"/> reference overlapping memory locations and do not begin at the same location.</exception>
+        /// <exception cref="ArgumentException"><paramref name="y"/> and <paramref name="destination"/> reference overlapping memory locations and do not begin at the same location.</exception>
+        /// <remarks>
+        /// <para>
+        /// This method effectively computes <c><paramref name="destination" />[i] = T.Ieee754Remainder(<paramref name="x" />[i], <paramref name="x" />[i])</c>.
+        /// </para>
+        /// </remarks>
+        public static void Ieee754Remainder<T>(ReadOnlySpan<T> x, ReadOnlySpan<T> y, Span<T> destination)
+            where T : IFloatingPointIeee754<T> =>
+            InvokeSpanSpanIntoSpan<T, Ieee754RemainderOperator<T>>(x, y, destination);
+
+        /// <summary>Computes the element-wise integer logarithm of numbers in the specified tensor.</summary>
+        /// <param name="x">The tensor, represented as a span.</param>
+        /// <param name="destination">The destination tensor, represented as a span.</param>
+        /// <exception cref="ArgumentException">Destination is too short.</exception>
+        /// <exception cref="ArgumentException"><paramref name="x"/> and <paramref name="destination"/> reference overlapping memory locations and do not begin at the same location.</exception>
+        /// <remarks>
+        /// <para>
+        /// This method effectively computes <c><paramref name="destination" />[i] = T.ILogB(<paramref name="x" />[i])</c>.
+        /// </para>
+        /// </remarks>
+        public static void ILogB<T>(ReadOnlySpan<T> x, Span<int> destination)
+            where T : IFloatingPointIeee754<T>
+        {
+            if (x.Length > destination.Length)
+            {
+                ThrowHelper.ThrowArgument_DestinationTooShort();
+            }
+
+            // TODO: Vectorize
+            for (int i = 0; i < x.Length; i++)
+            {
+                destination[i] = T.ILogB(x[i]);
+            }
```

Comment:
Any reason this isn't using `InvokeSpanIntoSpan<T, ILogBOperator<T>>(x, destination)` and then reporting `IsVectorizable => false`?

Would be a bit less churn for when I do get the vectorization support added.

(few others in this file with the same consideration)

Reply:
Because right now InvokeSpanIntoSpan requires that the destination be a `Span<T>`, not a `Span<int>` as this one requires.